_data/payments.yml

```diff
@@ -204,6 +204,8 @@ websites:
   bch: false
   btc: true
   othercrypto: true
+  exceptions:
+    text: GloBee used to accept BCH, but on 2018-11-23 stopped accepting it
```

Comment:
not certain but is that a specail character comma?

Reply:
My hex editor says 0x2c, normal comma